### PR TITLE
Bug/rtmd 8 remove back links

### PR DESCRIPTION
**Pull request by bendiggle**
Removed backlinks from the form.
- Removed from the steps.js 
- Removed from controllers where backLink is being set
- Removed backLink acceptance tests
- Added cancel button to add-additional-report (required due to removing backLinks)
- Added acceptance tests for cancel buttons (required due to removing backLinks)

### CODE REVIEW

File: apps/rtm/controllers/add-report.js [hunk 1]
```diff
@@ -13,7 +13,6 @@ module.exports = class AddReportController extends Controller {
     var reports = req.sessionModel.get('report') || [];
     if (reports.length) {
       lcls['additional-report'] = true;
```
Comment (JoeChapman) [inline apps/rtm/controllers/add-report.js]: still a shame about this var name

File: apps/rtm/controllers/add-report.js [hunk 2]
```diff
@@ -13,7 +13,6 @@ module.exports = class AddReportController extends Controller {
     var reports = req.sessionModel.get('report') || [];
```
Comment (JoeChapman) [inline apps/rtm/controllers/add-report.js]: probably out of scope of this pr, but as part of all our ongoing work, we should try and update to ES6

File: apps/rtm/controllers/add-report.js (hunk 1 above)
Comment (bendiggle) [inline apps/rtm/controllers/add-report.js]: In another PR

File: apps/rtm/controllers/add-report.js (hunk 2 above)
Comment (joefitter) [inline apps/rtm/controllers/add-report.js]: I think it make sense to have a dedicated ticket for doing this, rather than little bits here and there

File: apps/rtm/controllers/boilerplate.js [hunk 3]
```diff
@@ -10,10 +10,6 @@ module.exports = class BoilerPlate extends Controller{
 
   getValues(req, res, next) {
     this.options.clearSession = false;
-    res.locals.backLink = '/';
-    if (this.getReports(req).length) {
-      res.locals.backLink = '/confirmation';
-    }
     super.getValues(req, res, next);
   }
 };
```
Comment (joefitter) [inline apps/rtm/controllers/boilerplate.js]: Not part of this PR but this is a funny name for a controller. Its clearing session on get, does it do anything else, is it just a start controller?

**Comment by joefitter** [comment]
this LGTM

**Comment by JoeChapman** [comment]
I'll allow it

**Comment by bendiggle** [comment]
Thanks friends!
